### PR TITLE
Updated with an ADOPTERS.md

### PR DESCRIPTION
**Pull request by rjsadow**
Added a simple ADOPTERS.md with some placeholders for companies that have been interested in adoption of hydrophone.

### CODE REVIEW

**Comment by rjsadow** [comment]
@smira I've included talos as a placeholder adopter, has talos gone further down the road of adopting Hydrophone? If so, would you be willing to provide a short description of talos for this document?

@xrstf I've included kubermatic as well. Same for you all, have you adopted Hydrophone and would you be willing to provide a short description of kubermatic for this document?

**Comment by smira** [comment]
> @smira I've included talos as a placeholder adopter, has talos gone further down the road of adopting Hydrophone? If so, would you be willing to provide a short description of talos for this document?

Yes, sure, I will provide a description, and yes, we do use hydrophone to run Kubernetes conformance in an automated way via Talos CLI tool `talosctl`. Thank you for such a good project!

**Comment by xrstf** [comment]
> Kubermatic uses Hydrophone to verify our product releases' compliance with the Kubernetes conformance program.

Do you need/want something longer/more detailled? We had no time yet to migrate our automation to hydrophone, but for the k8s-conformance certification we use Hydrophone (manually). Replacing sonobuoy is still in our backlog.

File: ADOPTERS.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Hydrophone Adopters
+
+A non-exhaustive list of hydrophone adopters is provided below.
+
+**[Talos Linux](https://www.talos.dev/)** - placeholder
```
Comment (smira) [inline ADOPTERS.md:5]: Talos Linux uses `hydrophone` to run Kubernetes integration tests in several ways:

* `talosctl health --run-e2e` runs a short Kubernetes integration test after running Talos health checks
* `talosctl conformance` allows to run full integration test suite both "fast" mode (non-Serial tests in parallel, and Serial tests sequentially), and "certified" mode producing a bundle suitable for certification purposes

This is used both in continuous integration and available to end-users to verify their cluster configuratiion.

**Comment by rjsadow** [comment]
Hey @smira @xrstf, I'm moreso looking for a short blurb on the company or projects that use hydrophone as opposed to the actual implimentation of how you're using it. ref: https://github.com/containerd/containerd/blob/main/ADOPTERS.md?plain=1

**Comment by smira** [comment]
sorry, sure:

```
 Talos Linux is Linux designed for Kubernetes – secure, immutable, and minimal. Talos Linux is using `hydrophone` to run Kubernetes end-to-end tests, including certified conformance tests.
```

**Comment by xrstf** [comment]
> Kubermatic empowers organizations worldwide to fully automate their Kubernetes and cloud native operations across multi-cloud, edge and on-prem.

**Comment by xrstf** [comment]
/lgtm

:tada: :taco:

**Pull request merged**